File: 2020/territoriali/download/statement/download.pas
var
    T, tt, N, F, C : longint;
    nf, nc: longint; (* scrivi in queste variabili le risposte *)

begin
    (*
    se preferisci leggere e scrivere da file
    ti basta decommentare le seguenti due righe:
    *)

    (*
    assign(input, 'input.txt'); reset(input);
    assign(output, 'output.txt'); rewrite(output);
    *)

    readln(T);

    for tt := 1 to T do begin
        readln(); (* ignora la riga vuota *)
        readln(N, F, C);

        (* RISOLVI QUI IL PROBLEMA!! *)

        writeln('Case #', tt, ': ', nf, ' ', nc);
    end;
end.

File: 2020/territoriali/download/statement/download.c
#include <stdio.h>

int main() {
    // se preferisci leggere e scrivere da file
    // ti basta decommentare le seguenti due righe:

    //freopen("input.txt", "r", stdin);
    //freopen("output.txt", "w", stdout);

    int T, t;
    scanf("%d", &T);

    for (t = 1; t <= T; t++) {
        int N, F, C;
        
        // scrivi in queste variabili la risposta
        int nf, nc;

        scanf("%d %d %d", &N, &F, &C);

        // RISOLVI QUI IL PROBLEMA!!

        printf("Case #%d: %d %d\n", t, nf, nc);
    }
}
